### ISSUE TITLE
Bootstrap like nav-bar

### ISSUE THREAD

**Issue by jayminpanchal** [marcosmoura/vue-material#401]
Thanks for the awesome library but i have some question I am new to this Vue-Material.

1. Can we integrate with the bootstrap?
2. How to achieve nav-bar like bootstrap. Bootstrap provides link in header and when the screen resizes it automatically converted to side nav can we achieve this things with this library? I have searched so much but could not find anything.

Thanks for help

**Comment by Morgul** [comment]
TLDR; the answer is 'kinda'.

So, the components you're looking for are here: 

* https://vuematerial.github.io/#/components/toolbar
* https://vuematerial.github.io/#/ui-elements/layout

In order to have the automatic collapsing, you need to do a little work, there's nothing built in to do it for you. You can use a `md-layout` inside of the toolbar, to layout it's contents. Then for the elements that you want to hide, you can set the `md-hide-{size}` property on, hiding them when the screen size is small. However, there's no way to only show something at smaller sizes, so you'll have to write your own css classes; here's an example:

```sass
.show-small {
    @media(min-width:960px)
    {
        display: none !important;
    }
}
```

Not that this is optimal, by any means, but it's doable (and I'm currently doing this for my vue material sites). I ended up stealing Bootstrap's `hidden-{size}-up` and `hidden-{size}-down` scss code... if anyone's interested, I'll throw a cleaned up version into a gist.

**Comment by jayminpanchal** [comment]
Thanks @Morgul for your response. can you provide simple example of auto collapsing? It will be very helpful for me.
